src/util/localStorageActions.ts

```diff
@@ -0,0 +1,17 @@
+import { LOCAL_STORAGE_ACTIONS } from "constants/global";
+
+export const handleLocalStorage = (action: LOCAL_STORAGE_ACTIONS, key: string, data?: string): string | null => {
```

Comment:
A function should only be responsible for 1 single action. I would recommend for them to be split into multiple. Also I would do the `JSON.stringify` and `JSON.parse` inside each function to set and get the item respectively.